### PR TITLE
fix: add data-testid for download field

### PR DESCRIPTION
**Pull request by jbalfantz**

### CODE REVIEW

File: src/LineItems.tsx [hunk 1]
```diff
@@ -386,7 +390,7 @@ function getStyles() {
       border-top: 1px solid ${borderColor};
     `,
     refundDescription: css`
-      borderTop: none !important;
+      bordertop: none !important;
```
Comment (rachel-ftw) [inline src/LineItems.tsx]: Is this supposed to be camelcase? or should it have a dash?

File: src/LineItems.tsx [hunk 2]
```diff
@@ -397,13 +401,14 @@ function getStyles() {
       "& + *": {
         border-top: 1px solid ${borderColor};
         margin-top: 0;
-      },
+      }
```
Comment (rachel-ftw) [inline src/LineItems.tsx]: These are kind of weird styling corrections throughout.

File: src/LineItems.tsx (hunk 1 above)
Comment (jbalfantz) [reply src/LineItems.tsx]: @rachel-ftw I'm not sure - my prettier kicked in and formatted a bunch of things on this page.  (Same with your next comment).  I changed lines 179-183 only.

File: src/LineItems.tsx (hunk 2 above)
Comment (jbalfantz) [reply src/LineItems.tsx]: oh yeah, the comma is dropped to the next line, wtf?

File: __tests__/PurchaserDetails.test.tsx [hunk 3]
```diff
@@ -18,7 +20,7 @@ describe("PurchaserDetails", () => {
 
   test("renders correctly", () => {
     const props = {
-      orderDate: new Date("2018-05-21T00:00:00.000Z"),
+      orderDate: new Date(2018, monthMay, 20),
```
Comment (jbalfantz) [inline __tests__/PurchaserDetails.test.tsx]: I'm simply changing the UTC date (which fails 1/3 of the hours in the day) to the local date, which will always succeed.

**Pull request merged**